src/main/resources/templates/in-app/placement-information/v1.0.0.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>Sponsorship</title>
+  </head>
+  <body>
+    <h1>In-App Notification</h1>
+    <h2>Subject</h2>
+    <th:block th:fragment="subject">Placement Information</th:block>
+    <h2>Content</h2>
+    <th:block th:fragment="content">
+      <p>
+        For your placement starting
+        <b>
+          <th:block th:text="${startDate} ? |${#temporals.format(startDate, 'dd MMMM yyyy')}| : _">
+            (start date missing)
+          </th:block>
+        </b> at
+        <b>
+          <th:block th:text="${not #strings.isEmpty(site)} ? ${site} : _">
+            (site name missing)
+          </th:block>
+        </b> in
+        <b>
+          <th:block th:text="${not #strings.isEmpty(specialty)} ? ${specialty} : _">
+            (speciality name missing)
+          </th:block>
+        </b>, please see the below information.
```

Comment:
In the screenshot on the ticket, there is an extra space before the ', please see the below...' - I'm not sure if they specialty needed to be trimmed or simply put the `</b> etc`. on the same line as the `</th:block>` to avoid that issue.